### PR TITLE
feat: add a small collection of requested features and fixes

### PR DESCRIPTION
**Pull request by flexiondotorg**

### CODE REVIEW

File: quickemu [hunk 1]
```diff
@@ -1814,27 +1827,16 @@ fi
 if [ -n "${VM}" ] && [ -e "${VM}" ]; then
     # shellcheck source=/dev/null
     source "${VM}"
-    if [ -z "${disk_img}" ]; then
-        echo "ERROR! No disk_img defined."
-        exit 1
-    fi
-
-    if [ -e "${disk_img}" ]; then
```
Comment (philclifford) [inline quickemu:1822]: It must be somewhere else still:
```
quickemu --vm alpine-latest.conf --kill
qemu-img: Could not open 'alpine-latest/disk.qcow2': Failed to get shared "write" lock
Is another process using the image [alpine-latest/disk.qcow2]?
VM already started!
Killing alpine-latest
 - alpine-latest (6798) killed.
```

**Review by philclifford — COMMENTED** [review]
- --kill works - but still seeing the qemu-img output :confused: 
- --offline works 

LGTM but I have uneasy feelings that more testing is needed (and I can't 'til tonight)

**Comment by philclifford** [comment]
```
 quickemu --vm alpine-latest.conf --monitor-cmd quit
qemu-img: Could not open 'alpine-latest/disk.qcow2': Failed to get shared "write" lock
Is another process using the image [alpine-latest/disk.qcow2]?
VM already started!
Processing line: ssh,22220
 - Sending:  quit
```

we send the monitor command now, and quit works OK

I think maybe an `apt update` has been though, because now system_powerdown does nothing (system_reset works still)

**Comment by popey** [comment]
```
    elif [ -e "${disk_img}" ]; then
        # Check there isn't already a process attached to the disk image.
        if ! ${QEMU_IMG} info "${disk_img}" >/dev/null; then
            echo "             Failed to get \"write\" lock. Is another process using the disk?"
            exit 1
```
This looks like the culprit for this:

```
./quickemu --vm ./ubuntu-24.04.conf --kill                                                                                
qemu-img: Could not open 'ubuntu-24.04/disk.qcow2': Failed to get shared "write" lock                                                                                                         
Is another process using the image [ubuntu-24.04/disk.qcow2]?                                                                                                                                 
VM already started!                                                                                                                                                                           
Killing ubuntu-24.04                                                                                                                                                                          
 - ubuntu-24.04 (1539308) killed.                                           
```

Agreed offline works though :+1: 
Good on the resolution output too!

**Comment by flexiondotorg** [comment]
@philclifford and @popey Thanks for the feedback, there was quite a few logic bombs in there. Please test this branch again and let me know how it goes.

**Review by philclifford — APPROVED** [review]
LGTM
New features all good and --monitor-cmd works normally again.
A little casual testing with assistance of Old Peculiar seems satisfactory. :crossed_fingers:

**Pull request merged**